setup.cfg

```diff
@@ -1,2 +1,9 @@
 [metadata]
 version = attr: pystac.version.__version__
+
+[tool:pytest]
+minversion = 6.0
+addopts = -ra -q
+testpaths =
+    tests
+asyncio_mode = auto
```

Comment:
I wonder if we should leave these changes for a future PR that more comprehensively adds support for `pytest`. I'm in support of moving to `pytest` in general, but given that we don't include it in `requirements-test.txt` it seems out of place to have configuration for it here.

Reply:
I think this config is all that needs to be included for pytest to work (and not even that, since this just a bit more optimal). I think it's a separate question as to whether we make it the default/recommended way to run the tests.